driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -642,6 +654,7 @@ pub mod tests {
                 &AccountState::with_balance_for(&orders),
                 Duration::from_secs(180)
             )
+            .wait()
```

Comment:
Cannot be `now_or_never` because `blocking::unblock` is not immediately ready.

Reply:
Thanks for letting me know.